packages/react/src/components/Authenticator/index.tsx

```diff
@@ -44,24 +45,24 @@ export function Authenticator({
           switch (true) {
             case state.matches('idle'):
```

Comment:
Should this one also be switched to `actorState`?

Reply:
No, and here's an inconvenience I find about this refactor: `idle` is not part of any actor, so it's caught in the parent machine. I'm considering having better helper function that abstracts this actor layer (or can this be improved high level?)

something like this:
```tsx
enum AuthUIState {
  SignIn: 'signIn',
  SignUp: 'signUp',
  // ...
}

const getAuthUIState = (State machineState): AuthUIState => {
  // returns AuthUIState.SignIn, AuthUIState.SignUp, ...
}
```